btree/btree.c

```diff
@@ -2332,8 +2332,8 @@ static void mem_tree_put(struct td *tree)
 		m0_rwlock_write_lock(&trees_lock);
 		M0_ASSERT(trees_loaded > 0);
 		i = tree - &trees[0];
-		array_offset = i / sizeof(trees_in_use[0]);
-		bit_offset_in_array = i % sizeof(trees_in_use[0]);
+		array_offset = i / (sizeof(trees_in_use[0]) * 8);
+		bit_offset_in_array = i % (sizeof(trees_in_use[0]) * 8);
```

Comment:
Good catch. This same change needs to be done in tree_get routine where the offset it calculated at line 2244 in this file.

Reply:
Oh yes. Thanks.